docs/UserGuide.md

```diff
@@ -172,6 +172,20 @@ Examples:
 * `list` followed by `listt 1` lists transactions for the 1st person in the address book.
 * `find Betsy` followed by `listt 1` lists the transactions for the 1st person in the results of the `find` command.
 
+### Deleting Transactions for a specified person: `deletet`
+
+Format: `deletet INDEX`
+
+* Deletes the transactions at the specified `INDEX` for the first person on the current list.
+* The index refers to the index number of the transaction.
+* This command is intended to be used in conjunction with listt
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `listt 1` followed by `deletet 1` deletes the first transaction for the 1st person in the address book.
+* `find Betsy` followed by `deletet 1` deletes the first transaction for the 1st person 
```

Comment:
I don't think this is a correct usage? `find Betsy` displays a person list, not a transaction list. Maybe it should be like `findt 1 [keywords]` and then you can delete the transactions of person 1 that contain `keywords`.

Reply:
Thanks for spotting this, I'll update the UG